templates/dump-s3.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+aws s3 sync  s3://s3-dalite.harvardx.harvard.edu/ local_s3/  --region us-west-2
```

Comment:
Should we make the S3 bucket and region ansible vars instead of hardcoding them?